scripts/simpasm

```diff
@@ -278,7 +278,7 @@ def simplify(logger, args, asm_input, asm_output=None):
         else:
             autogen_header += [
                 f".global {sym}",
-                f"{sym}:",
+                f"{sym.replace('MLK_ASM_NAMESPACE', 'MLK_ASM_FN_SYMBOL')}",
```

Comment:
Why don't we use the macro in the dev files too?

Reply:
Or wait, we seem to... so why is this needed?

Reply:
The symbol name is grepped out of the .global directive. 
Before we were using MLK_ASM_NAMESPACE() for both the .global and the label. 
Now we use `MLK_ASM_NAMESPACE` for the .global, but `MLK_ASM_FN_SYMBOL` for the label. 
Would be nice to clean this up in the future and combine everything into one macro, but I did not manage to get that to work.

Reply:
Ack